test/core/util/port_isolated_runtime_environment.cc

```diff
@@ -43,14 +43,26 @@ static int get_random_port_offset() {
 static int s_initial_offset = get_random_port_offset();
 static gpr_atm s_pick_counter = 0;
 
-int grpc_pick_unused_port_or_die(void) {
+static int pick_unused_port_or_die(void) {
```

Comment:
Can you rename this to `grpc_pick_unused_port_or_die_impl` like the same function in port.cc.

Reply:
done.